diff --git a/build-support/bin/_release_helper.py b/build-support/bin/_release_helper.py
index 8a40d1ffa..990b67573 100644
--- a/build-support/bin/_release_helper.py
+++ b/build-support/bin/_release_helper.py
@@ -470,7 +470,7 @@ class _Constants:

     @property
     def pants_unstable_version(self) -> str:
-        return f"{self.pants_stable_version}+git{self._head_sha[:8]}"
+        return f"{self.pants_stable_version}"

     @property
     def twine_venv_dir(self) -> Path:
